### PR TITLE
Update/prevent trailing slash on api call with params

### PR DESCRIPTION
**Pull request by bryancodazen**
Rebase with fork origin and fixed a bug where a trailing slash on API calls with params is not working in the test environment.

I have updated the documentation here to reflect the changes: https://codazen.atlassian.net/wiki/spaces/FWD/pages/2549121027/Web+Diplomacy+Environment+information 

Screen Shot 2022-05-06 at 12 55 00 PM

VS

Screen Shot 2022-05-06 at 12 57 03 PM

### CODE REVIEW

File: beta-src/src/utils/api/index.ts [hunk 1]
```diff
@@ -26,10 +26,10 @@ export const getGameApiRequest = (
   route: ApiRoute,
   queryParams: QueryParams,
 ): Promise<AxiosResponse> =>
-  api.get(`?route=${route}&${buildQueryString(queryParams)}`);
+  api.get(`api.php?route=${route}&${buildQueryString(queryParams)}`);
```
Comment (mariomelchor) [inline beta-src/src/utils/api/index.ts:29]: shouldn't these be `/api.php` the env variable is `REACT_APP_WD_BASE_URL=http://localhost` without the slash at the end.

File: beta-src/src/utils/api/index.ts [hunk 2]
```diff
@@ -41,9 +41,9 @@ export const submitOrders = (
 ): Promise<AxiosResponse> => {
   if (Object.keys(queryParams).length) {
     return orderSubmission.post(
-      `/ajax.php?${buildQueryString(queryParams)}`,
+      `ajax.php?${buildQueryString(queryParams)}`,
```
Comment (mariomelchor) [inline beta-src/src/utils/api/index.ts:44]: same should these be `/ajax.php`

**Review by mariomelchor — APPROVED** [review]
A question otherwise LGTM!

File: beta-src/src/utils/api/index.ts (hunk 1 above)
Comment (bryancodazen) [reply beta-src/src/utils/api/index.ts:29]: No because axios automatically adds the trailing slash

File: beta-src/src/utils/api/index.ts (hunk 2 above)
Comment (bryancodazen) [reply beta-src/src/utils/api/index.ts:44]: No because axios automatically adds the trailing slash

**Comment by mounir-semaan** [comment]
LGTM :)

**Pull request merged**